src/main/java/teammates/common/datatransfer/StudentAttributesFactory.java

```diff
@@ -8,107 +8,104 @@
     private int commentColumnIndex;
-    
+
     private boolean hasSection;
     private boolean hasTeam;
     private boolean hasName;
     private boolean hasEmail;
     private boolean hasComment;
-        
+
     public StudentAttributesFactory() throws EnrollException {
         this("section|team|name|email|comment");
     }
-    
+
     /**
      * Construct a StudentAttributesFactory object with the given header row.<br>
      * The column names allowed in header row: {team, name, email, comment}<br>
-     * They are case insensitive and plural nouns are allowed.
-     * If the header row doesn't contain all required columns, then it is ignored and
-     * treated as a normal enroll line.
-     * <br>Pre-condition: headerRow must not be null
+     * They are case insensitive and plural nouns are allowed. If the header row
+     * doesn't contain all required columns, then it is ignored and treated as a
+     * normal enroll line. <br>
+     * Pre-condition: headerRow must not be null
+     * 
      * @throws EnrollException
      */
-    public StudentAttributesFactory(String headerRow) throws EnrollException {    
+    public StudentAttributesFactory(String headerRow) throws EnrollException {
         Assumption.assertNotNull(headerRow);
-        
+
         int fieldCount = locateColumnIndexes(headerRow);
-        
+
         if (fieldCount < MIN_FIELD_COUNT || !hasTeam || !hasName || !hasEmail) {
-            
             String missingField = "";
-            missingField = hasTeam? missingField : missingField + " <mark>Team</mark>";
-            missingField = hasName? missingField : missingField + " <mark>Name</mark>";
-            missingField = hasEmail? missingField : missingField + " <mark>Email</mark>";
+            missingField = hasTeam ? missingField : missingField + " <mark>Team</mark>";
+            missingField = hasName ? missingField : missingField + " <mark>Name</mark>";
+            missingField = hasEmail ? missingField : missingField + " <mark>Email</mark>";
             throw new EnrollException(ERROR_HEADER_ROW_FIELD_MISSED + ": " + missingField);
-        } 
+        }
     }
-    
+
     /**
      * Return a StudentAttributes object created from the given enrollLine.
+     * 
      * @throws EnrollException
      */
     public StudentAttributes makeStudent(String enrollLine, String courseId)
             throws EnrollException {
-        
         if (enrollLine.isEmpty()) {
             throw new EnrollException(ERROR_ENROLL_LINE_EMPTY);
         }
 
         String[] columns = splitLineIntoColumns(enrollLine);
 
-        boolean hasMissingFields = columns.length <= teamColumnIndex 
-                                || columns.length <= nameColumnIndex 
-                                || columns.length <= emailColumnIndex;
+        boolean hasMissingFields = columns.length <= teamColumnIndex
+                || columns.length <= nameColumnIndex
+                || columns.length <= emailColumnIndex;
```

Comment:
In the original code, the three conditions are aligned with each other and therefore easier to read.